### ISSUE TITLE
loader-utils need to update

### ISSUE THREAD

**Issue by morlay** [peerigon/markdown-loader#18]
```
(node:99162) DeprecationWarning: loaderUtils.parseQuery() received a non-string value which can be problematic, see https://github.com/webpack/loader-utils/issues/56
parseQuery() will be replaced with getOptions() in the next major version of loader-utils.
```

**Comment by meaku** [comment]
Thanks for the info. Looking into it :)

**Comment by eejdoowad** [comment]
Any updates on this?

**Comment by jannikkeye** [comment]
@eejdoowad looks like this has been updated: 7ac161e

**Comment by meaku** [comment]
Will publish asap.

**Comment by meaku** [comment]
Published as 2.0.1 https://www.npmjs.com/package/markdown-loader
Thanks @kommander!
